main.go

```diff
@@ -75,6 +86,28 @@ func main() {
 			instanceID = id
 		}
 
+		if cloudwatchStream == "" {
+			cloudwatchStream = instanceID
+		}
+
+		if cloudwatchGroup != "" {
+			hook, err := logrus_cloudwatchlogs.NewHook(cloudwatchGroup, cloudwatchStream, aws.NewConfig())
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			log.Infof("Writing logs to Cloudwatch Group %s, Stream %s", cloudwatchGroup, cloudwatchStream)
+			log.AddHook(hook)
+			log.SetOutput(ioutil.Discard)
```

Comment:
I think we should still log to to `stdout` so that the logs are available locally as well. This could help users debug network issues, IAM permissions etc?